include/lmms_constants.h

```diff
@@ -25,36 +25,54 @@
 #ifndef LMMS_CONSTANTS_H
 #define LMMS_CONSTANTS_H
 
-namespace lmms
```

Comment:
FYI, The C++ 20 PR was merged, so might as well make the switch now.

Reply:
Not entirely: https://github.com/LMMS/lmms/pull/7558#issuecomment-2495622443. Concepts and mathematical constants are waiting on GCC 10+. Looks like that is planned for after #7459 (see https://github.com/LMMS/lmms/pull/7558#issuecomment-2495678199), but I made these changes with the intention that they won't be dependent on the compiler upgrade.

Reply:
Ok then.